src-tauri/src/embed.rs

```diff
@@ -1,11 +1,10 @@
 #[tauri::command(async)]
 pub async fn embed(url: String) -> String {
```

Comment:
I should cleanup this command later.
Instead of just returning the string only return the HEAD element.
And maybe even do some Regex on the rust end already.

Reply:
i made #29 for this